en/deploy-on-aws-eks.md

```diff
@@ -36,30 +38,85 @@ apiVersion: eksctl.io/v1alpha5
 kind: ClusterConfig
 metadata:
   name: ${clusterName}
-  region: us-west-2
+  region: ap-northeast-1
 
 nodeGroups:
   - name: admin
     desiredCapacity: 1
+    privateNetworking: true
     labels:
       dedicated: admin
 
-  - name: tidb
-    desiredCapacity: 2
+  - name: tidb-1a
```

Comment:
TiDB doesn't have to be deployed 3 replicas.

Reply:
Updated